### DISCUSSION TITLE
Extracting derived parameters

### DISCUSSION THREAD

**Discussion by jabienias** [hpparvi/PyTransit#157] [Q&A] [answered]
I'm sorry if this seems a simple question, but I cannot see how to obtain the derived parameter values after running the model (a, i, t14 and t23), other than estimating them from the corner diagram.

Any help gratefully received.

John

**Comment by hpparvi** [comment, marked as answer]
Hi John,

Sorry, I should have documented this better. At the moment, you first get an xarray `DataSet` containing the posterior samples as

    df = ta.posterior_samples()

after which you can access the model parameter posterior samples as

    df.posterior.b_1

and the derived parameter posterior samples as

    df.derived_parameters.t14_1

Hope this helps!

Cheers,
Hannu

**Reply by jabienias** [reply]
That's fixed it! Many thanks.
John
